.NET/Microsoft.Recognizers.Text.DateTime/Arabic/Extractors/ArabicDateTimeAltExtractorConfiguration.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using Microsoft.Recognizers.Definitions.Arabic;
+
+namespace Microsoft.Recognizers.Text.DateTime.Arabic
+{
+    public class ArabicDateTimeAltExtractorConfiguration : BaseDateTimeOptionsConfiguration, IDateTimeAltExtractorConfiguration
```

Comment:
This sub-type has lower priority. Feel free to only work on it once all others are in place.

Reply:
you mean we should add tests and specs also for DateTimeModelComplexCalendar and DateTimeModelExtendedTypes? (They seem to be the ones using BaseDateTimeAltExtractor)
If so, what about the other models (CalendarMode, ExperimentalMode, SplitDateAndTime), should we add them too?

Reply:
Ideally yes, but they are even lower priority because they don't introduce new types. Thanks for catching this.